pyvista/__init__.py

```diff
@@ -89,11 +83,8 @@
 send_errors_to_logging()
 
 # Set preferred plot theme
-try:
-    theme = os.environ['PYVISTA_PLOT_THEME'].lower()
-    set_plot_theme(theme)
-except KeyError:
-    pass
+if 'PYVISTA_PLOT_THEME' in os.environ:
+    set_plot_theme(os.environ['PYVISTA_PLOT_THEME'].lower())
 
 # Set a parameter to control default print format for floats
 FLOAT_FORMAT = "{:.3e}"
```

Comment:
Might this be a candidate to be moved inside a theme? I guess FLOAT_FORMAT setss the format for legend in a plot?

Reply:
Agreed.  I'll resolve this.

Reply:
Turns out this is an internal parameter when printing out floats in our `__repr__`.  I'm going to leave it here.